rust/signed_doc/src/validator/rules/category.rs

```diff
@@ -1,180 +0,0 @@
-//! `content-type` rule type impl.
```

Comment:
Why have we eliminated this document type?
Its a valid document type and we would have to just reintroduce it later?

Reply:
No no, we have not removed the category document type.
Here what you are looking into is an old `category_id` field validation rule.
Which becomes a `parameters` validation rule inside its own mod.